### PR TITLE
Laud Misc 610

### PR DESCRIPTION
**Pull request by Eystein-Thanisch**
This data is from the entire text of CGH from Laud Misc 610 (Meyer's _Laud Genealogies and Tribal Histories_). 

Two new predicates were created, with ontologies, while encoding this data; see [bd3e3e1](https://github.com/cyocum/irish-gen/commit/bd3e3e1d9241a1b0649dd3646f422d115d590a07)

### CODE REVIEW

File: Laud_Misc_610/CGH/senchus_síl_hérimóin_annso_fo_héirind.trig [hunk 1]
```diff
@@ -0,0 +1,703 @@
...
+
+    a dctype:Dataset;
+    dcterms:title "Section 42: Senchus Síl hÉrimóin annso fo hÉirind"@sga;
+    dcterms:identifier "LMisc610_CGH_42";
+    dcterms:isFormatOf <https://celt.ucc.ie//published/G105005>;
+    dcterms:format "application/trig" ;
+    prov:asDerivedFrom <https://celt.ucc.ie//published/G105005> .
+
+<#Herimon>
+    a irishTitle:rí;
```
Comment (cyocum) [inline Laud_Misc_610/CGH/senchus_síl_hérimóin_annso_fo_héirind.trig]: `irishTitle:Rí;` search and replace :)

File: Laud_Misc_610/CGH/senchus_síl_hír.trig [hunk 2]
```diff
@@ -0,0 +1,1363 @@
...
+
+<#Ulaid>
+    a irishRel:populationGroup;
+    irishRel:populationGroupName "Ulaid";
+    irishRel:ancestorGroupOf <#DálFíatach>, <#DálAraide>;
+    owl:sameAs <http://example.com/Rawl_B502/section_11.trig#Ulaid>;
+    rdfs:comment "Cóic ríg fichet do hUltaib rogabsat ríge nÉirenn cenmothát na secht ríg rogabsat do Dál Fíatach, dáig is do chloind Óengusa Turbig Temrach & Úgaine Móir do Dáil Fhiatach.".
+
+<#DálFíatach>
+    a irishRel:populationGroup;
```
Comment (cyocum) [inline Laud_Misc_610/CGH/senchus_síl_hír.trig]: I don't remember if I mentioned this in this file but `irishRel:PopulationGroup` search and replace :)

File: Laud_Misc_610/CGH/senchus_síl_hír.trig [hunk 3]
```diff
@@ -0,0 +1,1363 @@
...
+    irishRel:genName "Fedlimthe";
+    irishRel:nomName "Feidlimid";
+    owl:sameAs <http://example.com/LL/clanna_ébir_i_l-leith_chuind.trig#Feidlimid>.
+
+_:missing-71c7d9a0
+    a irishTitle:rí;
+    rel:childOf <#Húamanchenn>.
+
+<#Dáre>
+    a irishTitle:rí;
```
Comment (cyocum) [inline Laud_Misc_610/CGH/senchus_síl_hír.trig]: `irishTitle:Rí` search and replace

File: LL/h_n_echdach.trig [hunk 4]
```diff
@@ -270,8 +260,12 @@ _:missing-d48779e1
 <#MaelodorCáech>
     a foaf:Person;
     irishRel:nomName "Maelodor Cáech";
-    rel:childOf <#Fheradaig-f8d424e0>;
-    rdfs:comment "a quo .h. Brótaid i Cluain Fiacna".
+    rel:childOf <#Fheradaig-f8d424e0>.
```
[the remark this answers is not in the document]
Comment (Eystein-Thanisch) [reply LL/h_n_echdach.trig]: Done

File: Laud_Misc_610/CGH/cland_conchuboir_iterum.trig [hunk 5]
```diff
@@ -0,0 +1,597 @@
...
+    rel:childOf <#hUislend>;
+    rdfs:seeAlso <#RombeLatheRorduRind>;
+    owl:sameAs <http://example.com/Laud_Misc_610/CGH/senchus_síl_hír.trig#Ardáin>.
+
+<#Fergus>
+    a foaf:Person;
+    irishRel:nomName "Fergus";
+    rel:childOf <#Róich>;
+    rel:childOf <#Róich-69f50610>;
+    rel:childOf <#Roích-a5320330>;
```
[the remark this answers is not in the document]
Comment (Eystein-Thanisch) [reply Laud_Misc_610/CGH/cland_conchuboir_iterum.trig]: I've condensed, as you suggest.

This is a bit of a strange situation. Fergus is given three alternative pedigrees. In each case, his father is "Ro-ech" but then they each have a different father. So I have made them different people.

File: Laud_Misc_610/CGH/de_causis_quibus_exules_aquilonensium_ad_muminenses_adducti_sunt.trig [hunk 6]
```diff
@@ -0,0 +1,200 @@
...
+    irishRel:genName "Rosa Róig";
+    irishRel:nomName "Ros Róig";
+    owl:sameAs <http://example.com/LL/fergus_mc_do_roig.trig#Roig>.
+
+<#MedbaCrúachan>
+    a foaf:Person;
+    foaf:gender "female";
+    irishRel:genName "Medba Crúachan";
+    irishRel:nomName "Medb Crúachan";
+    rel:spouseOf <#Fergusa>;
```
[the remark this answers is not in the document]
Comment (Eystein-Thanisch) [reply Laud_Misc_610/CGH/de_causis_quibus_exules_aquilonensium_ad_muminenses_adducti_sunt.trig]: You're right. A "lover" predicate would actually be really useful. Can we create it quickly just now?

I love how family-friendly all the off-the-peg relationships are (spouse, children, employer etc.)!

File: Laud_Misc_610/CGH/cland_conchuboir_iterum.trig (hunk 5 above)
[the remark this answers is not in the document]
Comment (cyocum) [reply Laud_Misc_610/CGH/cland_conchuboir_iterum.trig]: That's cool

File: Laud_Misc_610/CGH/de_causis_quibus_exules_aquilonensium_ad_muminenses_adducti_sunt.trig (hunk 6 above)
[the remark this answers is not in the document]
Comment (cyocum) [reply Laud_Misc_610/CGH/de_causis_quibus_exules_aquilonensium_ad_muminenses_adducti_sunt.trig]: Ahah, yeah. Literature and history are less family-friendly :P

File: Laud_Misc_610/CGH/de_causis_quibus_exules_aquilonensium_ad_muminenses_adducti_sunt.trig [hunk 7]
```diff
@@ -0,0 +1,200 @@
...
+    irishRel:populationGroupName "mocu Chíara".
+
+<#Conaillamedbmichura>
+    dcterms:title "Conailla medb michura";
+    prov:wasAttributedTo <#Lucrith>;
+    owl:sameAs <http://vanhamel.nl/codecs/Conailla_Medb_m%C3%ADchuru>.
+
+# This section is very terse and the connections between characters are not always apparent.
+# It should be reviewed against the IrishGen database and other sources once more material has been added.
+# Also, a translation exists of 'Conailla medb michura' in Études Celtiques 29; this will be
```
[the remark this answers is not in the document]
Comment (Eystein-Thanisch) [reply Laud_Misc_610/CGH/de_causis_quibus_exules_aquilonensium_ad_muminenses_adducti_sunt.trig]: Would that that were so. Unfortunately, _Études Celtiques_ is a different journal and it isn't online.

File: Laud_Misc_610/CGH/do_cheniul_suibne.trig [hunk 8]
```diff
@@ -0,0 +1,505 @@
...
+
+<#Óengus>
+    a foaf:Person;
+    irishRel:nomName "Óengus";
+    rel:childOf <#Maelcobai>;
+    irishRel:ancestorOfGroup <#CenélnÓengusa>;
+    owl:sameAs <http://example.com/LL/do_cheniul_shuibne.trig#Oengus>.
+
+<#CenélnÓengusa>
+    a irishRel:populationGroup;
```
[the remark this answers is not in the document]
Comment (Eystein-Thanisch) [reply Laud_Misc_610/CGH/do_cheniul_suibne.trig]: Changed throughout the corpus. I included the semi-colon in the Find so as to only change it in this context.

File: Laud_Misc_610/CGH/genelach_cloindi_conchubair_chorraig.trig [hunk 9]
```diff
@@ -0,0 +1,727 @@
...
+    a irishRel:populationGroup;
+    irishRel:populationGroupName "Síl Cathasaig".
+
+<#hOiDacennan>
+    a irishRel:populationGroup;
+    irishRel:populationGroupName "hOi Dacennan";
+    owl:sameAs <http://example.com/LL/clainde_conchobuir_corraig.trig#h-ÓDacennan>.
+
+<#Echen-c3c641b0>
+    a foaf:person;
```
[the remark this answers is not in the document]
Comment (Eystein-Thanisch) [reply Laud_Misc_610/CGH/genelach_cloindi_conchubair_chorraig.trig]: Fixed

File: Laud_Misc_610/CGH/genelach_cloindi_conchubair_chorraig.trig [hunk 10]
```diff
@@ -0,0 +1,727 @@
...
+    irishRel:nomName "Echen";
+    rel:childOf <#Echan>;
+    owl:sameAs <http://example.com/LL/clainde_conchobuir_corraig.trig#Echen-1ba55679>;
+    irishRel:ancestorOfGroup <#hUíMaicCárthinn>, <#hUíChonaing>, <#CenélnEchen>.
+
+<#Echan>
+    a foaf:person;
```
[the remark this answers is not in the document]
Comment (Eystein-Thanisch) [reply Laud_Misc_610/CGH/genelach_cloindi_conchubair_chorraig.trig]: Fixed

File: Laud_Misc_610/CGH/genelach_cloindi_conchubair_chorraig.trig [hunk 11]
```diff
@@ -0,0 +1,727 @@
...
+    irishRel:nomName "Echen";
+    irishRel:genName "Echan";
+    owl:sameAs <#Echen-cd369e10>.
+
+<#hUíMaicCárthinn>
+    a irishRel:populationGroup;
```
[the remark this answers is not in the document]
Comment (Eystein-Thanisch) [reply Laud_Misc_610/CGH/genelach_cloindi_conchubair_chorraig.trig]: Fixed in mass change

File: Laud_Misc_610/CGH/de_causis_quibus_exules_aquilonensium_ad_muminenses_adducti_sunt.trig (hunk 7 above)
[the remark this answers is not in the document]
Comment (cyocum) [reply Laud_Misc_610/CGH/de_causis_quibus_exules_aquilonensium_ad_muminenses_adducti_sunt.trig]: Arg! I always get those two mixed up!

File: Laud_Misc_610/CGH/scéla_eogain_in_so_&_cormaic.ttl [hunk 12]
```diff
@@ -0,0 +1,147 @@
...
+    dcterms:identifier "LMisc610_CGH_16";
+    dcterms:isFormatOf <https://celt.ucc.ie//published/G105005>;
+    dcterms:format "application/trig" ;
+    prov:asDerivedFrom <https://celt.ucc.ie//published/G105005> .
+
+<#EnganMór>
+    a foaf:Person;
+    irishRel:nomName "Engan Mór";
+    owl:sameAs <http://example.com/Laud_Misc_610/CGH/do_minigud_senchais_fer_muman.trig#Eogan>.
+# Possible typo? EPT
```
[the remark this answers is not in the document]
Comment (Eystein-Thanisch) [reply Laud_Misc_610/CGH/scéla_eogain_in_so_&_cormaic.ttl]: Thanks! Corrected this here and where referenced elsewhere.

File: Laud_Misc_610/CGH/senchas_airgíall_in_so.ttl [hunk 13]
```diff
@@ -0,0 +1,675 @@
...
+
+    a dctype:Dataset;
+    dcterms:title "Section 19: Senchas Airgíall in so"@sga;
+    dcterms:identifier "LMisc610_CGH_19";
+    dcterms:isFormatOf <https://celt.ucc.ie//published/G105005>;
+    dcterms:format "application/trig" ;
+    prov:asDerivedFrom <https://celt.ucc.ie//published/G105005> .
+
+<#hAirgíalli>
+    a irishRel:populationGroup;
```
[the remark this answers is not in the document]
Comment (Eystein-Thanisch) [reply Laud_Misc_610/CGH/senchas_airgíall_in_so.ttl]: Fixed in mass change

File: Laud_Misc_610/CGH/senchas_airgíall_in_so.ttl [hunk 14]
```diff
@@ -0,0 +1,675 @@
...
+
+<#Inhomomon>
+    a foaf:Person;
+    irishRel:nomName "Inhomomon";
+    rel:childOf <#MugdornDub>;
+    irishRel:ancestorOfGroup <#CorcuInmendi>;
+    owl:sameAs <http://example.com/LL/h_airgialla.trig#Inhomomon>.
+
+<#CorcuInmendi>
+    a irishRel:populationGroup;
```
[the remark this answers is not in the document]
Comment (Eystein-Thanisch) [reply Laud_Misc_610/CGH/senchas_airgíall_in_so.ttl]: Done

File: Laud_Misc_610/CGH/senchus_dáil_fíatach.trig [hunk 15]
```diff
@@ -0,0 +1,823 @@
...
+    a foaf:Person;
+    irishRel:nomName "Daig";
+    rel:childOf <#Dubthaig>;
+    owl:sameAs <http://example.com/LL/senchas_dáil_fhiatach.trig#Daig>;
+    owl:sameAs <http://example.com/Laud_Misc_610/CGH/senchus_síl_hír.trig#Daig>;
+    rdfs:comment "Is de atá Carnd Dega";
+    irishRel:ancestorOfGroup <#hÚiDega>.
+
+<#hÚiDega>
+    a irishRel:populationGroup;
```
[the remark this answers is not in the document]
Comment (Eystein-Thanisch) [reply Laud_Misc_610/CGH/senchus_dáil_fíatach.trig]: Done ;-)

File: Laud_Misc_610/CGH/senchus_síl_hérimóin_annso_fo_héirind.trig (hunk 1 above)
Comment (Eystein-Thanisch) [reply Laud_Misc_610/CGH/senchus_síl_hérimóin_annso_fo_héirind.trig]: Done

File: Laud_Misc_610/CGH/senchus_síl_hír.trig (hunk 2 above)
Comment (Eystein-Thanisch) [reply Laud_Misc_610/CGH/senchus_síl_hír.trig]: Oh yes. Thanks for the reminder. I hadn't picked up on that suggestion :-D

File: Laud_Misc_610/CGH/senchus_síl_hír.trig (hunk 3 above)
Comment (Eystein-Thanisch) [reply Laud_Misc_610/CGH/senchus_síl_hír.trig]: Done

**Pull request merged**